controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -229,8 +224,31 @@ func (r *StorageClassClaimReconciler) reconcileConsumerPhases() (reconcile.Resul
 			sccEncryptionMethod := r.storageClassClaim.Spec.EncryptionMethod
 			_, scIsFSType := existing.Parameters["fsName"]
 			scEncryptionMethod, scHasEncryptionMethod := existing.Parameters["encryptionMethod"]
-			if !((sccType == "sharedfilesystem" && scIsFSType && !scHasEncryptionMethod) ||
-				(sccType == "blockpool" && !scIsFSType && sccEncryptionMethod != scEncryptionMethod)) {
+
+			validStorageClassConfig := true
```

Comment:
change this to `invalidStorageclass:=false` for better readability?

Reply:
I'm ready to change, tagging @nb-ohad to add his thought, just to make we are on the same page